docker/1.6.0/py3/Dockerfile.gpu

```diff
@@ -42,11 +46,60 @@ RUN apt-get update \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
 
+# Install cuda-dev libraries as a dependency for Horovod with MXNet backend
 RUN wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/libcublas10_10.2.1.243-1_amd64.deb \
+    https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/libcublas-dev_10.2.1.243-1_amd64.deb \
```

Comment:
For safety in terms of managing these links, we should probably add a new ARG to save this url base and construct based on it

Reply:
Do you mean something like?

```
ARG CUDA_LIB_URL=https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64
RUN wget ${CUDA_LIB_URL}/libcublas10_10.2.1.243-1_amd64.deb \
         ${CUDA_LIB_URL}/libcublas-dev_10.2.1.243-1_amd64.deb \
```

Reply:
yeah exactly

Reply:
Okay. We generally use `ARG` if we want to change the arguments during build time, such as `docker build -t MXNet:py3-gpu-build --build-arg CUDA_LIB_URL=/some/url .`. Why do we want to change this URL during build process?

Reply:
it's a good point -- likely wouldn't want to change during build process, using an ENV is fine too

Reply:
Done [5c4b7c](https://github.com/aws/sagemaker-mxnet-container/pull/133/commits/5c4b7c6ca77afd9b976cdd16fd7497053a14435d)